java/src/test/java/ai/rapids/cudf/RmmTest.java

```diff
@@ -399,6 +399,14 @@ public void testPoolLimitNonPoolMode() {
         () -> Rmm.initialize(RmmAllocationMode.CUDA_DEFAULT, false, 1024, 2048));
   }
 
+  @Test
```

Comment:
please update the copyright in this file to 2021.

Reply:
Fixed. Thanks!